subprojects/code-quality/src/integTest/groovy/org/gradle/api/plugins/quality/pmd/PmdPluginAuxclasspathIntegrationTest.groovy

```diff
@@ -70,7 +70,7 @@ class PmdPluginAuxclasspathIntegrationTest extends AbstractPmdPluginVersionInteg
                 }
 
                 pmd {
-                    ruleSets = ["java-auxclasspath"]
+                    ruleSets = ["rulesets/java/auxclasspath.xml"]
```

Comment:
why did this change?

Reply:
New deprecation in 6.46.0: <https://pmd.sourceforge.io/pmd-6.46.0/pmd_release_notes.html#deprecated-ruleset-references>